NSS-Day-Cohort-66/rare-client-extra-medium-1
# Create Category

# Categories can now be created via POST method & can be reached using "Create New Category" button on Category Manager

**New Supported Route**
- `/create-category`

## HOW TO TEST
1. Pull down 'NM-client4' branch and git checkout
2. Go to Category manager
3.  Click 'Add New Category' button
4. Add new category
5. Ensure the new category was added to the list on Category Manager
